cmd/crc/cmd/root.go

```diff
@@ -170,6 +163,9 @@ func addForceFlag(cmd *cobra.Command) {
 
 func executeWithLogging(fullCmd string, input func(cmd *cobra.Command, args []string) error) func(cmd *cobra.Command, args []string) error {
 	return func(cmd *cobra.Command, args []string) error {
+		for _, str := range defaultVersion().lines() {
+			logging.Debugf(strings.TrimSpace(str))
+		}
```

Comment:
No idea why it's better there?

Reply:
Yes, not obvious for me either, giving up this.